README.adoc

```diff
@@ -189,49 +189,49 @@ Configure the default base URL with the MicroProfile Config feature. This featur
-[source, properties, linenums, role='code_column']
+[source, properties, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/src/main/webapp/META-INF/microprofile-config.properties[tags=**;!copyright]
+include::finish/src/main/webapp/META-INF/microprofile-config.properties[]
 ----
 
-The config property [hotspot=1 file=0]`io.openliberty.guides.inventory.client.SystemClient/mp-rest/url` is configured to the default [hotspot=2 file=0]`\http://localhost:9080/system` base URL.
+The config property [hotspot=1 file=0]`io.openliberty.guides.inventory.client.SystemClient/mp-rest/url` is configured to the default [hotspot=Config file=0]`\http://localhost:9080/system` base URL.
 
 This configuration is automatically picked up by the MicroProfile Config API.
 
-Look at the annotations in the [hotspot=17-22 file=1]`SystemClient` interface again.
+Look at the annotations in the [hotspot=SystemClient file=1]`SystemClient` interface again.
 
 SystemClient.java
-[source, Java, linenums, role='code_column']
+[source, Java, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/src/main/java/io/openliberty/guides/inventory/client/SystemClient.java[tags=**;!copyright]
+include::finish/src/main/java/io/openliberty/guides/inventory/client/SystemClient.java[]
 ----
 
-The [hotspot=13 file=1]`@Dependent` annotation tells the service that the scope of the interface depends on the class that it is injected into.
+The [hotspot=Dependent file=1]`@Dependent` annotation tells the service that the scope of the interface depends on the class that it is injected into.
 By default, interfaces have a `@Dependent` scope unless another scope is defined on the interface.
 
-The [hotspot=14 file=1]`@RegisterRestClient` annotation registers the interface as a RESTful client. The runtime creates a CDI managed bean for every interface that is annotated with the `@RegisterRestClient` annotation.
+The [hotspot=RegisterRestClient file=1]`@RegisterRestClient` annotation registers the interface as a RESTful client. The runtime creates a CDI managed bean for every interface that is annotated with the `@RegisterRestClient` annotation.
 
-The [hotspot=13 file=1]`@Dependent` and [hotspot=14 file=1]`@RegisterRestClient` annotations imply that the interface is manageable through CDI. You must have them in order to inject the client.
+The [hotspot=Dependent file=1]`@Dependent` and [hotspot=RegisterRestClient file=1]`@RegisterRestClient` annotations imply that the interface is manageable through CDI. You must have them in order to inject the client.
 
-Inject the [hotspot=17-22 file=1]`SystemClient` interface into the [hotspot=25-102 file=2]`InventoryManager` class, which is another CDI managed bean. 
+Inject the [hotspot=SystemClient file=1]`SystemClient` interface into the [hotspot=InventoryManager file=2]`InventoryManager` class, which is another CDI managed bean. 
 
 [role="code_command hotspot", subs="quotes"]
 ----
 #Create the `InventoryManager` class.#
```

Comment:
This `Create` block highlights the `microprofile-config.properties` file instead of the `InventoryManager.java` file.

Reply:
Opened a separate issue since README looks correct (also an issue on master).